tests/selinux.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "selinux test" {
+  if ! which selinuxenabled ; then
+    skip
+   elif ! /usr/sbin/selinuxenabled; then
+     skip
```

Comment:
These two lines are indented inconsistently.

Reply:
Fixed

Reply:
Might be useful to add a message to `skip` (e.g. `skip "couldn't find selinuxenabled"`) indicating why the test is being skipped.